db/models.py

```diff
@@ -1,35 +1,48 @@
+from typing import Any
 from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.conf import settings
+from django.core.exceptions import ValidationError
 
 
 class Genre(models.Model):
-    name = models.CharField(max_length=255, unique=True)
+    name: str = models.CharField(max_length=255, unique=True)
 
     def __str__(self) -> str:
         return self.name
 
 
 class Actor(models.Model):
-    first_name = models.CharField(max_length=255)
-    last_name = models.CharField(max_length=255)
+    first_name: str = models.CharField(max_length=255)
+    last_name: str = models.CharField(max_length=255)
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
 
 
 class Movie(models.Model):
-    title = models.CharField(max_length=255)
-    description = models.TextField()
-    actors = models.ManyToManyField(to=Actor, related_name="movies")
-    genres = models.ManyToManyField(to=Genre, related_name="movies")
+    title: str = models.CharField(max_length=255)
+    description: str = models.TextField()
```

Comment:
Type annotations for Django model fields are not necessary and could lead to confusion. It's better to use Django's field types directly without type annotations.